EmpleoDotNet/Views/Shared/_JobListPartial.cshtml

```diff
@@ -9,7 +9,7 @@
             <a href="@Url.Action("Detail", "JobOpportunity" , new { id=UrlHelperExtensions.SeoUrl(jobOpportunity.Id, jobOpportunity.Title)})" class="featured">
                 <div class="row">
                     <div class="col-md-1 hidden-sm hidden-xs text-center">
-                        @if (!string.IsNullOrEmpty(jobOpportunity.CompanyLogoUrl))
+                        @if (UrlHelperExtensions.IsImageAvailable(jobOpportunity.CompanyLogoUrl))
```

Comment:
Thanks, @anyeloamt, and what about the image url that aren't working right now, can we make a query to set those fields to null, so It will show the Initials and not the broken image icon?